extension/src/popup/views/SignTransaction/index.tsx

```diff
@@ -144,6 +144,12 @@ export const SignTransaction = () => {
   const isMemoRequired = flaggedKeyValues.some(
     ({ tags }) => tags.includes(TRANSACTION_WARNING.memoRequired) && !memo,
   );
+  const isUnsafe = flaggedKeyValues.some(({ tags }) =>
+    tags.includes(TRANSACTION_WARNING.unsafe),
+  );
+  const isMaliciousAddress = flaggedKeyValues.some(({ tags }) =>
+    tags.includes(TRANSACTION_WARNING.malicious),
+  );
```

Comment:
I mistakenly removed this in an earlier PR thinking we'd use Blockaid for this use case. These flags are actually if the address if malicious (not the asset). Since Blockaid doesn't offer address scanning for Stellar yet, we should continue using stellar.expert to scan malicious addresses